scripts/deploy-remote.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -e
+
+BACKEND_ARTIFACT="/tmp/roombaht-backend.tgz"
+FRONTEND_ARTIFACT="/tmp/roombaht-frontend.tgz"
+
+BACKEND_DIR="/opt/roombaht-backend"
+FRONTEND_DIR="/opt/roombaht-frontend"
+
+ENV_FILE="/tmp/secrets.env"
+
+problems() {
+    2>&1 echo "Error: $*"
+    exit 1
+}
+
+cleanup() {
+    rm "$ENV_FILE" "$BACKEND_ARTIFACT" "$FRONTEND_ARTIFACT"
+}
+
+[ -e "$BACKEND_ARTIFACT" ] || problems "unable to find backend artifact"
+[ -e "$FRONTEND_ARTIFACT" ] || problems "unable to find frontend artifact"
+[ -e "$ENV_FILE" ] || problems "unable to find env file"
+
+source /tmp/secrets.env
+export ROOMBAHT_SECRET_KEY
```

Comment:
I'm unclear on this action here. Sourcing the secrets file will load the env vars from secrets into the current shell instance. These export lines are doing what then? Confirming the var exists?

Reply:
The `secrets.env` file just has the raw variables, they are not exported. The variables which are required for things the deploy script uses are then exported for use.